config/config.go

```diff
@@ -34,6 +34,9 @@ const (
 	SettingJWTIssuer        = "jwt_issuer"
 	SettingJWTIssuerDefault = "mender.useradm"
 
+	SettingAllowedJWTIssuers        = "allowed_jwt_issuers"
+	SettingAllowedJWTIssuersDefault = ""
```

Comment:
I think the config needs documentation in the `config.yaml`

Reply:
fixed.